fedora/readme.md

```diff
@@ -0,0 +1,42 @@
+# Guide
+## to create rpm package and update distribution package
+
+### General Information
+
+These instructions assume, that you have most/all of the tools needed for the compilation of software generally and qesteidutil specifically already installed. For the dependencies of qesteidutil, see the main Readme of the repository.
```

Comment:
It does not hurt to give a copy-paste line for installing deps

Reply:
Should be there now.